specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/bastionHost.json

```diff
@@ -404,6 +625,219 @@
         }
       },
       "description": "Response for ListBastionHosts API service call."
+    },
+    "BastionShareableLink": {
+      "properties": {
+        "vm": {
```

Comment:
can this be called 'vmid' and just use the resource id? Its currently modeled as needing all /resource properties which you likely don't need. For get/list, the user may not have permission to see the VM properties either.